crates/bevy_render/src/camera/camera.rs

```diff
@@ -41,7 +44,11 @@ impl Camera {
         camera_transform: &GlobalTransform,
         world_position: Vec3,
     ) -> Option<Vec2> {
-        let window = windows.get(self.window)?;
+        // FIXME: just a hack to make this compile!
+        // I suspect we do need a window here, or perhaps the Viewport should also store scale.
+        // I also suspect this function is only relevant for Windows and not for offscreeen
+        // textures.
```

Comment:
I think it would be relevant for off-screen textures. In that case it should go from world_position to pixel coordinate on the texture I think.

Reply:
I just realized that we do want a viewport here, for when the window is split into multiple viewports. So we definitely need to store scale, which would then be 1.0 for the off-screen case.

Reply:
I think this is addressed in my second attempt, thanks!